infra/apim_v2/apim_pnpg.tf

```diff
@@ -153,8 +153,8 @@ module "apim_pnpg_external_api_ms_v2" {
   api_operation_policies = [
     {
       operation_id = "getInstitutionsUsingGET"
-      xml_content = templatefile("./api_pnpg/external_api_for_pnpg/v2/getInstitutions_op_policy.xml.tpl", {
-        BACKEND_BASE_URL           = "https://selc-${var.env_short}-ext-api-backend-ca.${var.ca_pnpg_suffix_dns_private_name}/v2/"
+      xml_content = templatefile("./api_pnpg/external_api_for_pnpg/v2/jwt_base_policy_product_subscription.xml.tpl", {
```

Comment:
you could use the common policy inside _/api_ folder, `/api/jwt_base_policy.xml.tpl ` in xml_contente above and `/api/base_ms_url_policy.xml ` here...look an example at module.apim_pnpg_support_service_v2 in the main branch.